docs/_config.yml

```diff
@@ -33,7 +33,7 @@ collections:
     sort_by: order
 
 # Build settings
-theme: just-the-docs
+# theme: just-the-docs
```

Comment:
Can we just delete it rather than leave it commented out.

Reply:
Removed.